tools/run_tests/run_xds_tests.py

```diff
@@ -1782,6 +1815,90 @@ def _delay(pct):
         set_validate_for_proxyless(gcp, True)
 
 
+def test_csds(gcp, original_backend_service, instance_group, server_uri):
+    logger.info('Running csds')
+
+    logger.info('waiting for original backends to become healthy')
+    wait_for_healthy_backends(gcp, original_backend_service, instance_group)
+
+    logger.info('start sending traffic')
+    configure_client(
+        rpc_types=[messages_pb2.ClientConfigureRequest.RpcType.UNARY_CALL])
```

Comment:
The client sends RPCs by default IIUC; I don't think you need this.

Reply:
Good catch. Removed.